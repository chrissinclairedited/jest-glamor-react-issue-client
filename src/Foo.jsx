import React from "react";
import { Button } from "library";

export default () => (<Button onClick={() => console.log('foo')}>Foo button</Button>);
