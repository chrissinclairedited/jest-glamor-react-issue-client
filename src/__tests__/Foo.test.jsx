import React from "react";
import { mount } from "enzyme";
import Foo from "../Foo";

describe("<Foo />", () => {
    it("does not show the styles for the imported components", () => {
        const component = mount(<Foo />);
        expect(component).toMatchSnapshot();
    })
})
