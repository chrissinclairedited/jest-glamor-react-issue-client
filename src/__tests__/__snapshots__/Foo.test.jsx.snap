// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Foo /> does not show the styles for the imported components 1`] = `
<Component>
  <Component
    onClick={[Function]}
  >
    <div
      data-css-1ea0ufb=""
    >
      <button
        onClick={[Function]}
      >
        Foo button
      </button>
    </div>
  </Component>
</Component>
`;
